#! /bin/sh /usr/share/dpatch/dpatch-run
## 40_additional_transform_fixes.dpatch by Henning Glawe <glaweh@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: additional fixes for transform.pd

@DPATCH@
diff -urNad pdl-2.4.2/Lib/Transform/transform.pd /tmp/dpep.pT50KV/pdl-2.4.2/Lib/Transform/transform.pd
--- pdl-2.4.2/Lib/Transform/transform.pd	2005-01-15 15:27:15.000000000 +0100
+++ /tmp/dpep.pT50KV/pdl-2.4.2/Lib/Transform/transform.pd	2005-01-15 15:39:35.000000000 +0100
@@ -1588,7 +1588,7 @@
 
 
   ### The first argument is a dummy to set $GENERIC.
-  $idx = PDL_Double($idx) unless($idx->type == PDL_Double);
+  $idx = double($idx) unless($idx->type == double);
   &PDL::_map_int( $in2->flat->index(0), 
 	$in2, $o2, $idx, 
 	$bound, $method, $big, $blur, $svmin, $flux );
